manifests/config/project.pp

```diff
@@ -55,7 +55,8 @@
   $node_executor_settings = {},
   $projects_dir           = undef,
   $resource_sources       = $rundeck::resource_sources,
-  $scm_import_properties  = $rundeck::scm_import_properties,
+  $scm_import_properties  = {},
```

Comment:
What I mean is that hiera's automatic lookups are known to fail on blank array defaults. Do they fail with blank hash defaults?